crates/biome_cli/tests/snapshots/main_cases_biome_json_support/check_biome_json.snap

```diff
@@ -27,7 +27,7 @@ debugger;
 # Emitted Messages
 
 ```block
-Fixed 1 file(s) in <TIME>
+Checked 1 file(s) in <TIME>; 1 file(s) are changed, 0 file(s) are unchanged.
```

Comment:
IMHO, I prefer not to display any information in the console, neither changed nor unchanged, when there are zero results. Because I think it is better to display only the necessary information.

e.g. 
- `Checked 1 file(s) in <TIME>; 1 file(s) are changed.`
- `Checked 1 file(s) in <TIME>; 1 file(s) are unchanged.`